lib/aryn-sdk/aryn_sdk/partition/partition.py

```diff
@@ -107,9 +107,19 @@ def partition_file(
 
     _logger.debug(f"{options_str}")
 
+    # Work around hanging behavior when vcrpy is recording...
```

Comment:
actually can you add more verbosity to the comment to describe why this is necessary

Reply:
maybe stick the details in an issue, link it and close it